scripts/liveness

```diff
@@ -41,20 +41,19 @@ demote() {
 }
 
 status() {
-	if [[ -f liveness-syncing ]]; then
+	R=`awk 'BEGIN { srand(); print int(rand()*32768) }' /dev/null`
```

Comment:
AS we are in the nitpick space here I must point out that bacticks are not recommneded. $(...) is preferred - there are some very old bourne style dialects that does not support them but in general the community don't care about that.

> Because of these inconsistent behaviors, the backquoted variety of command substitution is not recommended for new applications that nest command substitutions or attempt to embed complex scripts.

At : https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_xcu_chap02.html#tag_23_02_06_03

See also http://mywiki.wooledge.org/BashFAQ/082

Reply:
Fixed!